test/helpers.js

```diff
@@ -11,7 +11,13 @@ if (process.env.API_INTEGRATION_TESTS) {
 
 const env = process.env !== undefined ? process.env : window.__env__
 
-export const initClient = () => testUtils.initClient()
+export const initClient = () => {
+  const accessToken = env.CONTENTFUL_INTEGRATION_TEST_CMA_TOKEN
+  return createClient({
+    accessToken,
+    ...params,
+  })
+}
```

Comment:
Is this a leftover change or what's the reason we swapped away from testUtils.initClient()

Reply:
`testUtils.initClient` does the same internally. The issue was that I couldn't test my changes because `testUtils` has this repo as dependency and the version they have doesn't yet have the changes i made, so when I tried to call `getAll` it had no args in the types

Reply:
Can you add a TODO here? Or should we just strip that out entirely from the repo? My fear is that we're going to drift and there will be some bad expectations

Reply:
Added a comment. I think it should do for now, but I share your concern and there is a high chance of it happening, so in my opinion, we should delete it from test-utils as well.

Reply:
But for that we need to research where else it is used to not break something on accident. If the same function from test-utils is used elsewhere, then I am afraid that this comment is all we can do